pkg/rhel/openvdc.spec

```diff
@@ -85,6 +87,13 @@ This is a 'stub'. An appropriate message must be substituted at some point.
 /opt/axsh/openvdc/bin/openvdc-executor
 %dir /etc/openvdc
 %config(noreplace) /etc/openvdc/executor.toml
+%config(noreplace) /etc/firewalld/services/vdc-executor.xml
+
+%post executor
+%{firewalld_reload}
```

Comment:
I don't think that the behavior. ``firewall-cmd reload`` is just like ``systemdctl daemon-reload``.

```
$ sudo firewall-cmd --get-default-zone
public
$ sudo firewall-cmd --zone=public --list-all
public (active)
  target: default
  icmp-block-inversion: no
  interfaces: eth0 eth1
  sources:
  services: dhcpv6-client ssh
  ports:
  protocols:
  masquerade: no
  forward-ports:
  sourceports:
  icmp-blocks:
  rich rules:
$ sudo vi /usr/lib/firewalld/services/vdc-executor.xml
$ sudo firewall-cmd --get-services | grep vdc-executor
$ sudo firewall-cmd --reload
success
$ sudo firewall-cmd --get-services | grep vdc-executor
RH-Satellite-6 amanda-client amanda-k5-client bacula bacula-client ceph ceph-mon dhcp dhcpv6 dhcpv6-client dns docker-registry dropbox-lansync freeipa-ldap freeipa-ldaps freeipa-replication ftp high-availability http https imap imaps ipp ipp-client ipsec iscsi-target kadmin kerberos kpasswd ldap ldaps libvirt libvirt-tls mdns mosh mountd ms-wbt mysql nfs ntp openvpn pmcd pmproxy pmwebapi pmwebapis pop3 pop3s postgresql privoxy proxy-dhcp ptp pulseaudio puppetmaster radius rpc-bind rsyncd samba samba-client sane smtp smtps snmp snmptrap squid ssh synergy syslog syslog-tls telnet tftp tftp-client tinc tor-socks transmission-client vdc-executor vdsm vnc-server wbem-https xmpp-bosh xmpp-client xmpp-local xmpp-server
$ sudo firewall-cmd --zone=public --list-all
public (active)
  target: default
  icmp-block-inversion: no
  interfaces: eth0 eth1
  sources:
  services: dhcpv6-client ssh
  ports:
  protocols:
  masquerade: no
  forward-ports:
  sourceports:
  icmp-blocks:
  rich rules:

```



```
$ rpm --showrc | grep firewall
-14: firewalld_reload
test -f %{_bindir}/firewall-cmd && firewall-cmd --reload --quiet || :
```